github_format.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -x
+
+echo "Copying workflows from cloudposse/.github repo"
+git config --local user.name "${BOT_NAME}"
+git config --local user.email "11232728+${BOT_NAME}@users.noreply.github.com"
+
+mkdir gha_tmp_dir
+cd gha_tmp_dir
+git clone https://github.com/cloudposse/.github
+cp ./.github/workflow-templates/*.yml ../.github/workflows/
+cd ..
+rm -rf ./gha_tmp_dir
```

Comment:
```suggestion
dir=$(mktemp -d)
git clone https://github.com/cloudposse/.github $dir
cp -a $dir/workflow-templates/*.yml .github/workflows/
rm -rf $dir
```

Reply:
Also, I cannot imagine a world where we want all `$dir/workflow-templates/*.yml` workflows, but instead select workflows.